src/components/ui/Table/index.tsx

```diff
@@ -26,7 +33,7 @@ TableItem.Wrapper = styled.div<Pick<ItemProps, 'flexValue' | 'textAlign'>>`
   padding-left: 2%;
   width: 100%;
 `;
-TableCard.Wrapper = styled.div`
+TableCard.Wrapper = styled.div<Pick<ItemProps, 'mouseCursor'>>`
   display: flex;
   height: 50px;
   background-color: ${SECONDARY_LIGHT_SMOKE};
```

Comment:
pixel value I thought we were using rem/em

Reply:
Oversight  👀